### ISSUE TITLE
Does ASW work with OpenXR?

### ISSUE THREAD

**Issue by timtaotao** [Oculus-VR/Unity-Graphics#26]
Hi, 
I want to use OpenXR as the plugin provider, and enable its Meta XR feature group. However, I failed to get ASW running. ASW is working correctly with Oculus, since I want to use certain OpenXR features, I am wondering if I can get ASW with OpenXR?
Thanks

**Comment by jonkelling** [comment]
Which OpenXR features in particular?

**Comment by timtaotao** [comment]
@jonkelling  Things like XR Hands etc.
Just wondering if I can enable AppSpaceWarp with OpenXR as the plugin provider, is this possible?

**Comment by jonkelling** [comment]
No, I don't believe you can do this right now. I did read that the Oculus XR plugin will eventually be deprecated in favor of the OpenXR plugin, but there does not appear to be a clear timeline on this.

**Comment by smitdylan2001** [comment]
I believe you van manually set AppSW now in OpenXR
